File: 3-4주차/B10989.js
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

const N = Number(input[0]);
const counting = new Array(10001).fill(0);

for (let i = 1; i <= N; i++) {
    const num = Number(input[i]);
    counting[num]++;
}

let output = "";
for (let i = 0; i < counting.length; i++) {
    for (let j = 0; j < counting[i]; j++) {
        output += i + "\n";
    }
}

process.stdout.write(output);

File: 3-4주차/B10989.c
#include <stdio.h>
 
int main(void){
    int N;
    scanf("%d", &N);
 
    int counting[10001];
    for(int i=0; i<10001; i++){
        counting[i] = 0;
    }
 
    for(int i=0; i<N; i++){
        int input;
        scanf("%d", &input);
        counting[input]++;
    }
 
    for(int i=0; i<10001; i++){
        for(int j=0; j<counting[i]; j++){   
            printf("%d\n", i);
        }
    }
}